src/protocol.rs

```diff
@@ -34,9 +34,8 @@ const OFFSET_FETCH_KEY: i16 = 9;
 const VERSION: i16 = 0;
 
 const FETCH_MAX_WAIT_TIME: i32 = 100;
-const FETCH_MIN_BYTES: i32 = 4096;
-const FETCH_BUFFER_SIZE_BYTES: i32 = 4096;
-const MAX_FETCH_BUFFER_SIZE_BYTES: i32 = FETCH_BUFFER_SIZE_BYTES * 8;
+const FETCH_MIN_BYTES: i32 = 1;
```

Comment:
I think it was a temporary test that eventually stuck around in the pull request :) https://github.com/frew/kafka-rust/commit/c5a0b5eff26de53c812b949c310af06b41d785d7